packages/frontend/core/src/blocksuite/presets/ai/chat-panel/components/add-popover.ts

```diff
@@ -93,9 +93,16 @@ export class ChatPanelAddPopover extends SignalWatcher(
   @property({ attribute: false })
   accessor abortController!: AbortController;
 
+  @query('.search-input')
+  accessor searchInput!: HTMLInputElement;
+
   override connectedCallback() {
     super.connectedCallback();
     this._updateDocGroup();
+
+    requestAnimationFrame(() => {
+      this.searchInput.focus();
+    });
```

Comment:
I think moving `focus` to `firstUpdated` is better
```suggestion
  }

  override firstUpdated() {
      this.searchInput.focus();
```

Reply:
You are right. Thanks! Please take another look.